public/styles/components/_minimap.css

```
@@ -0,0 +1,179 @@
+.minimap-container.visible {
+  opacity: 1;
+  transform: scale(1);
+}
+
+/* Hover effect (for debugging) */
+.minimap-container:hover {
+  transform: scale(1.05);
+}
+
+/* High DPI screen support */
+@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 2dppx) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)